doc/ref/chap-type-method.md

```diff
@@ -963,9 +963,30 @@ If the command fails, `captureStdout()` raises an error, which can be caught
 with `try`.
 
     try {
-      var s = _io->captureStdout(c)
+      var s = io.captureStdout(c)
     }
 
+### captureOutputs()
+
+Capture stdout and stderr of a command a string.
+
+    var c = ^(echo hi; echo hello >&2;)
+    var outputs = _io.captureOutputs(c)  # => { stdout: "hi", stderr: "hello", status: 0 }
+
+It's like `io.captureStdout` but captures both stdout and stderr.  Trailing newlines `\n` are
```

Comment:
We may want to keep trailing newlines?

The only reason captureStdout() does it is because that's how shell command sub work

I also wonder if there are better names for `captureOutputs()` ...  I was thinking `captureStdoutStderr()`, but that leaves out `status`

or `capture3()` or something

Reply:
I thought especially of `capture3`, too. But it's IMO a bit too "bound to the output". E.g. if we ever want to add something like `additional_fds=[3, 4]`, or execution time, etc. Therefore `Outputs` seemed the cleanest.

Reply:
I also thought it could be `captureStd()` or `captureAll()`

I feel like captureStdout() + captureOutputs() is a tiny bit awkward, because is "stderr" and "status" an "output"?  It is hard to find a good name though

I guess All() can also refer to the fact that it doesn't strip newlines

Reply:
hm `captureAll` sounds good! Let me change it to that and fix the remaining points.